packages/rspack/tests/copyPlugin/build/main.js

```diff
@@ -1 +1 @@
-(function(){var t={435:function(t,e,r){"use strict";r.r(e);var o=r("287");e.default=o},287:function(t,e,r){"use strict";t.exports=r.p+"asset-modules/deepnested.txt"}},e={};function r(o){var n=e[o];if(void 0!==n)return n.exports;var i=e[o]={exports:{}};return t[o](i,i.exports,r),i.exports}r.r=function(t){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(t,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(t,"__esModule",{value:!0})},r.g=function(){if("object"==typeof globalThis)return globalThis;try{return this||Function("return this")()}catch(t){if("object"==typeof window)return window}}(),!function(){r.g.importScripts&&(t=r.g.location+"");var t,e=r.g.document;if(!t&&e&&(e.currentScript&&(t=e.currentScript.src),!t)){var o=e.getElementsByTagName("script");if(o.length){for(var n=o.length-1;n>-1&&!t;)t=o[n--].src}}if(!t)throw Error("Automatic publicPath is not supported in this browser");t=t.replace(/#.*$/,"").replace(/\?.*$/,"").replace(/\/[^\/]+$/,"/"),r.p=t}(),r("435")})();
\ No newline at end of file
+!function(){var t={435:function(t,e,r){"use strict";r.r(e);var o=r("287");e.default=o},287:function(t,e,r){"use strict";t.exports=r.p+"asset-modules/deepnested.txt"}},e={};function r(o){var n=e[o];if(void 0!==n)return n.exports;var i=e[o]={exports:{}};return t[o](i,i.exports,r),i.exports}r.r=function(t){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(t,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(t,"__esModule",{value:!0})},r.g=function(){if("object"==typeof globalThis)return globalThis;try{return this||Function("return this")()}catch(t){if("object"==typeof window)return window}}(),!function(){r.g.importScripts&&(t=r.g.location+"");var t,e=r.g.document;if(!t&&e&&(e.currentScript&&(t=e.currentScript.src),!t)){var o=e.getElementsByTagName("script");if(o.length){for(var n=o.length-1;n>-1&&!t;)t=o[n--].src}}if(!t)throw Error("Automatic publicPath is not supported in this browser");t=t.replace(/#.*$/,"").replace(/\?.*$/,"").replace(/\/[^\/]+$/,"/"),r.p=t}(),r("435")}();
```

Comment:
this seems change the value of iife？

Reply:
This is a bugfix. 
Previously, due to the absence of a parenthesis remover, many assumptions within the minifier were invalid. 
After fixing the bug, more minifier rules are applied. 

In this case, the return value of the IIFE is not used, and this transformation only occurs under this condition.

Reply:
it will cause problem when the code is eval scenario, so is a swc option like  negate_iife in terser to control this behavior?

Reply:
> is a swc option like negate_iife in terser to control this behavior?

Yes.

With [negate_iife: true](https://play.swc.rs/?version=1.4.2&code=H4sIAAAAAAAAA9NIK81LLsnMz1NIy8%2FX0FSo5lIAgrT8IgWNCmsFEIKJ1XLVampoAgB7VYOLMAAAAA%3D%3D&config=H4sIAAAAAAAAA12PSw6DMAxE95wCed1FlUUXvUPPUFmpQanIR7GRihB3rwmBom6i%2BGVmPJmbtoU3W7i3s151SJiZ8jEr4SkIfpQAWY9ss0sC5XG5bB7B3JMUBZurMVD5ECOT4g4Hpsq8C66bzvkeQz%2F8yRTb6FMm5pNUaaAehZ7OdatD8ngYlmY%2FSy3w8TWW2PoxmRJtFW%2FwE%2B1tjt3g%2BLE71%2Fhm%2BQJkYWsMJAEAAA%3D%3D)
With [negate_iife: false](https://play.swc.rs/?version=1.4.2&code=H4sIAAAAAAAAA9NIK81LLsnMz1NIy8%2FX0FSo5lIAgrT8IgWNCmsFEIKJ1XLVampoAgB7VYOLMAAAAA%3D%3D&config=H4sIAAAAAAAAA12PPQrDMAyF95zCaO5QPHToHXqGIlwluMQ%2FWAo0hNy9wnHS0EVIn96Tn5fOGHizg7tZtNUhY2Eqx6yE5yj4UQLkArIrPgvU5XrZPIJlIKkKtldrofExJSbFPY5MjQUffT%2Bf7weMw%2FgnU%2BxSyIWYT1KlkQYUenrfH462XLu91lgQ0muqZ9vHZM60RbzBT7SnOd4Gz4%2FdKWWibv0C5z49jyQBAAA%3D)

Reply:
then it's fine